### ISSUE TITLE
Don't hard-code tab behavior

### ISSUE THREAD

**Issue by pihart** [cjquines/qboard#191]
And preferably don't even bind to tab; this makes the app horribly inaccessible

**Comment by cjquines** [comment]
the app was inaccessible as is anyway? like, if youre navigating with keyboard how are you going to even draw right. i dont think this is an issue at all

**Comment by pihart** [comment]
I reasonably agree with this (that's why I hadn't mentioned the concern until now) but the whole motto of qboard is to the effect of "everything should be doable with your keyboard." And you can't even set keybindings or open the bindings modal (unless you happen to have the `1`/`?` binding memorized) with your keyboard!!! This seems fundamentally flawed and i think we should reasonably deal with it.

(And keyboard-only use is becoming increasingly more plausible given that TeXing can be done entirely via keyboard. This isn't the reason for this issue, though.)
